pepper-apis/docs/specification/src/endpoints/user.profile.addresses.verify.yml

```diff
@@ -12,9 +12,62 @@ post:
   tags:
     - Mailing Address
   requestBody:
-    $ref: '../pepper.yml#/components/requestBodies/MailingAddress'
+    required: true
+    content:
+      application/json:
+        schema:
+          allOf:
+            - $ref: '../pepper.yml#/components/schemas/BaseMailAddress'
+            - type: object
+              properties:
+                studyGuid:
+                  type: string
+                  description: if provided, will check address against study rules for any potential issues
   responses:
     200:
-      $ref: '../pepper.yml#/components/responses/MailingAddress'
+      description: a corrected address that's better suited for deliverability, along with any warnings
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '../pepper.yml#/components/schemas/BaseMailAddress'
+              - type: object
+                required:
+                  - studyGuid
+                  - warnings
```

Comment:
Looks something like this:

![Screen Shot 2020-05-26 at 2 54 54 PM](https://user-images.githubusercontent.com/5713833/82941459-6186ed80-9f64-11ea-8509-e06386f1301d.png)